cms/djangoapps/contentstore/models.py

```diff
@@ -21,3 +21,28 @@ def get_profile_whitelist(cls):
 
 class PushNotificationConfig(ConfigurationModel):
     """Configuration for mobile push notifications."""
+
+
+class MigrateVerifiedTrackCohortsSetting(ConfigurationModel):
+    """
+    ...
```

Comment:
Empty comment block.

We will have to remember to go back and delete this stuff once all courses have completed transitioning. Actually, we should create a JIRA ticket for the cleanup, including the original verifiedtrackcohorting code.